lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -495,7 +525,12 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
         this.ngZone.run(() => {
             this.resetSelection();
             if (this.node) {
-                this.data.loadPage(this.node, this._pagination.merge);
+                if (this.hasPreSelectedNodes()) {
+                    this.data.loadPage(this.node, this._pagination.merge, null, this.preSelectedNodes);
+                    this.onNodeSelect({ row: <ShareDataRow> this.data.getPreSelectedRows()[0], selection: <ShareDataRow[]> this.data.getPreSelectedRows() });
```

Comment:
if the intension is to bubble up the selection, I don't think we need it ...

Reply:
We need to bubble up only preselectedNodes selection